Source/Particles/SpeciesPhysicalProperties.H

```diff
@@ -55,7 +57,8 @@ namespace species
         }
     }
 
-    static amrex::Real get_mass (PhysicalSpecies ps)
+    AMREX_FORCE_INLINE
+    amrex::Real get_mass (PhysicalSpecies ps)
```

Comment:
Fixes the warning:
`./Source/Particles/SpeciesPhysicalProperties.H:58:24`: warning: ‘amrex::Real species::get_mass(PhysicalSpecies)’ defined but not used [-Wunused-function]
     static amrex::Real get_mass (PhysicalSpecies ps)